### ISSUE TITLE
Export benchmarked data into json

### ISSUE THREAD

**Issue by JuanitoFatas** [JuanitoFatas/fast-ruby#19]

**Comment by nurettin** [comment]
This could be used as an extension to RubyMine that provides context-sensitive tips for better performance

**Comment by JuanitoFatas** [comment]
> This could be used as an extension to RubyMine that provides context-sensitive tips for better performance

Nice! How do I create these tips for RubyMine?

**Comment by JuanitoFatas** [comment]
[benchmark-ips now can export json](https://github.com/evanphx/benchmark-ips/pull/29).

**Comment by mateusdeap** [comment]
So, I'm not sure what should be done here. Maybe we export this json with our test runs?

An alternative is to create a command that exports this data so that other libs can possibly call it.

Finally, this issue has been around for so long without conclusion that maybe it's not worth the trouble? At least for now. I like the motivation, which is why I think it'd be interesting to expose some sort of command that could just export all the benchmarks into a json file.

What do you think, @ashwinisukale?

**Comment by ashwinisukale** [comment]
@mateusdeap agree, idea is nice, need to check if we already have any export functionality and if we have then we can add one more option or command in it to get the data.

**Comment by inbdra** [comment]
Oke

Pada tanggal Sel, 11 Okt 2022 15.48, Ashwini Sukale <
***@***.***> menulis:
